### PR TITLE
issue-92: remove @Sneaky-Throws from tests dev branch

### PR DESCRIPTION
**Pull request by EEM86**
Remove @SneakyThrows from tests

### CODE REVIEW

**Review by Segelzwerg — DISMISSED** [review]

**Comment by Segelzwerg** [comment]
@EEM86 I think you mixed up the branches during commit.

**Comment by EEM86** [comment]
> 
> 
> @EEM86 I think you mixed up the branches during commit.

@Segelzwerg Hm, I had not done it. Let's see. I saw the last commit on develop branch on 10/10/2020 #eedd and further my commit with small changes.

**Comment by Segelzwerg** [comment]
> > @EEM86 I think you mixed up the branches during commit.
> 
> @Segelzwerg Hm, I had not done it. Let's see. I saw the last commit on develop branch on 10/10/2020 #eedd and further my commit with small changes.

Yeah I did a hotfix for the CI, but I think the `toString()` is out of scope here and you could do that in the PR for #87

**Comment by Segelzwerg** [comment]
> > > @EEM86 I think you mixed up the branches during commit.
> > 
> > 
> > @Segelzwerg Hm, I had not done it. Let's see. I saw the last commit on develop branch on 10/10/2020 #eedd and further my commit with small changes.
> 
> Yeah I did a hotfix for the CI, but I think the `toString()` is out of scope here and you could do that in the PR for #87

Oh I get it. You merged your stuff in your develop. So it is merged here too.

**Comment by EEM86** [comment]
> 
> 
> > > > @EEM86 I think you mixed up the branches during commit.
> > > 
> > > 
> > > @Segelzwerg Hm, I had not done it. Let's see. I saw the last commit on develop branch on 10/10/2020 #eedd and further my commit with small changes.
> > 
> > 
> > Yeah I did a hotfix for the CI, but I think the `toString()` is out of scope here and you could do that in the PR for #87
> 
> Oh I get it. You merged your stuff in your develop. So it is merged here too.

Ah oh, I saw that you merged my previous commit and just pulled the develop branch and pushed new commit with #95 task.

**Comment by Segelzwerg** [comment]
> > > > > @EEM86 I think you mixed up the branches during commit.
> > > > 
> > > > 
> > > > @Segelzwerg Hm, I had not done it. Let's see. I saw the last commit on develop branch on 10/10/2020 #eedd and further my commit with small changes.
> > > 
> > > 
> > > Yeah I did a hotfix for the CI, but I think the `toString()` is out of scope here and you could do that in the PR for #87
> > 
> > 
> > Oh I get it. You merged your stuff in your develop. So it is merged here too.
> 
> Ah oh, I saw that you merged my previous commit and just pulled the develop branch and pushed new commit with #95 task.

Yeah, I merged the CI stuff, because I noticed it not working. I think I have to do some more on it.

File: src/main/java/segelzwerg/sporttooolbox/iunits/distance/Miles.java [hunk 1]
```diff
@@ -89,4 +89,9 @@ public Kilometer toKilometer() {
     public Time computeTime(Speed speed) {
         return speed.computeTime(miles, yards);
     }
+
+    @Override
+    public String toString() {
+        return String.format("%d mile and %d yard", miles, yards);
+    }
```
Comment (Segelzwerg) [inline src/main/java/segelzwerg/sporttooolbox/iunits/distance/Miles.java]:
```suggestion
```

File: src/main/java/segelzwerg/sporttooolbox/iunits/distance/Kilometer.java [hunk 2]
```diff
@@ -80,6 +80,6 @@ public Kilometer addDistance(Distance distance) {
 
     @Override
     public String toString() {
-        return kilometer + " kilometer and " + meter + " meter";
+        return String.format("%d kilometer and %d meter", kilometer, meter);
```
Comment (Segelzwerg) [inline src/main/java/segelzwerg/sporttooolbox/iunits/distance/Kilometer.java]:
```suggestion
       return kilometer + " kilometer and " + meter + " meter";
```

File: src/main/java/segelzwerg/sporttooolbox/iunits/distance/Nautical.java [hunk 3]
```diff
@@ -100,4 +100,9 @@ public Time computeTime(Speed speed) {
     private float getNautical() {
         return nautical + fathoms / NAUTICAL_TO_FATHOMS;
     }
+
+    @Override
+    public String toString() {
+        return String.format("%d nautical and %.2f fathom", nautical, fathoms);
+    }
```
Comment (Segelzwerg) [inline src/main/java/segelzwerg/sporttooolbox/iunits/distance/Nautical.java]:
```suggestion

```

**Review by Segelzwerg — CHANGES_REQUESTED** [review]
Can you revert the `toString()` stuff as it will decrease coverage and is out of scope in this PR?

**Comment by EEM86** [comment]
> 
> 
> Can you revert the `toString()` stuff as it will decrease coverage and is out of scope in this PR?

Sure. One minute, please.

**Pull request merged**